django/dkg/cancer/json_api/endpoints.py

```diff
@@ -0,0 +1,72 @@
+import json # need double quotes in json POST bodies
+from django.http import JsonResponse
+from cancer.models import TrainingArticle, InferenceArticle
+
+
+def train(request):
+    """
+    Receives a POST-request with the following body: ?
+
+    Stores the data only.
+
+    :param request: HTTP-request carrying all RIS article information.
+    :param persistence: persistence object.
+    :return: possible HTTP-status codes: 200, 400.
+    """
+    request_body = json.loads(request.body)
+    print request_body
+    # store all parsed articles
+    article = TrainingArticle(
+        title='some article',
+        abstract='abstract of article',
+        authors='a\tb',
+        keywords='4,harn-prog\t1,harn-pall'
+    )
+    article.save()
+
+    return JsonResponse({'foo': 'bar'})
+
+
+def inference(request):
+    """
+    Receives a POST-request with the following body: ?
+    and runs multi-label inference on it.
+
+    :param request: HTTP-request carrying all RIS articles (without keywords)
+    :param persistence: persistence object.
+    :return: json response with all of the predicted labels from the uploaded RIS articles.
+
+    Response body sample: ?
+    """
+    request_body = json.loads(request.body)
+    print request_body
+    # store all parsed articles for inference
+    article = InferenceArticle(
```

Comment:
In InferenceArticle werden die Felder aus TrainArticle einfach kopiert oder? Kann man auch einfach sowas wie InferenceArticle(article=TrainingArticle, label='some label') machen? oder geht das wegen der django db datenmodell einschraenkungen nich? 

irgendwie kommts mir komisch vor, zwei objekte fuer fast dasselbe zu haben, alternativ koennte man ja auch einfach Article(title=...) haben und label bei den test articles erstmal leer lassen? 

eigentlich aber auch egal. 

wichtiger ist wohl: wir werden ja mehrere feedbacks/labels pro article bekommen - wie kann man das in dem setting darstellen? 

ich hab auch mal dran gedacht, einfach nur Articles zu haben, und dann so 'label' objekte, also alle ein feedback event, dass dann user-id enthaelt, und ne article id, mit der man das auf nen article (der noch kein label hat) matchen kann. so koennte man dann eine komponente haben, die etwa nen 'majority vote' aus mehreren labels errechnet.

aber man kanns auch genau so wie's is machen - nur wie man mehrere labels handled is mir noch nich klar. is vielleicht auch nich so wichtig, der workflow der annotators is eh so, dass sich immer ueber conflicting labels verstaendigt wird, bis es nur noch ein label gibt.